x-pack/plugins/apm/public/components/routing/templates/backend_detail_template.tsx

```diff
@@ -46,9 +67,35 @@ export function BackendDetailTemplate({ title, children }: Props) {
 
   const { data: { metadata } = {} } = backendMetadataFetch;
 
+  const tabs = isOperationsBreakdownFeatureEnabled
```

Comment:
I think in this case that's a little harder to do because we need context to determine the value of the feature flag. In the case that you're referring to everything's static. Does that make sense?